docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -478,7 +478,7 @@ Post-Release Review Process
 
 #. Verify that the client (``flocker-deploy``) can be installed on all supported platforms:
 
-   Follow the :ref:`Flocker client installation documentation<installing-flocker-cli>`.
+   Follow the Flocker client installation documentation at https://docs.clusterhq.com/en/${VERSION}/indepth/installation.html#installing-flocker-cli.
```

Comment:
It would be different for marketing and development releases. I don't have a strong preference either way - if you do, I'll change it.